plutus-core-spec/figures/TypeSynthesis.tex

```diff
@@ -202,12 +202,13 @@
 
     \begin{prooftree}
         \alwaysNoLine
-        \AxiomC{$bn$ has signature $\sig{\alpha_0 :: K_0, ..., \alpha_m :: K_m}{B_0, ..., B_n}{C}$ in Figure \ref{fig:builtins}}
+        \AxiomC{$bn$ has signature $\sig{\alpha_1 :: K_1, ..., \alpha_k :: K_k}{B_1, ..., B_n}{C}$ in Figure \ref{fig:builtins}}
+        \UnaryInfC{\(\hypJ{\Gamma}{\istypeJ{A_i}{K_i}}\)}
```

Comment:
Do we want everything to be indexed by `i` here?  It kind of makes it looks as if things have to match up that actually don't have to.

Reply:
I think that's important, no? It's showing that there's a typing hypothesis for all each individual argument.

Reply:
Yeah, I'm just not quite sure what the convention is, like whether you regard the things on each line as being independent from each other, and whether all of the indices are implicitly universally quantified.  Paging @jmchapman !